tests/integration/test_parameter_runner.py

```diff
@@ -15,6 +16,13 @@ def test_parameter_runner(experiment_to_run: experiment.SingleParameterExperimen
         experiment_to_run.runs_per_scenario = (
             1  # Likely only need to test the one run per scenario
         )
+
+        experiment_to_run.scenario.price_process_fn = lambda: random_walk(
+            num_steps=experiment_to_run.scenario.num_steps,
+            starting_price=1000,
+            sigma=0.1,
+        )
```

Comment:
Hmm, I'm not sure whether we want to blindly override every scenario with the same random walk config ultimately, but it's perhaps ok for now. We should probably at some point separate out the concept of testing the parameter runners work and testing specific scenarios in any case

Reply:
I agree, I think we probably need one test per scenario, and then maybe two tests for the parameter runner (network parameter test and market parameter test).